dashboard/test/ui/features/pd/teacherApplication.feature

```diff
@@ -0,0 +1,90 @@
+   And I press the first "input[name='race'][value='Other']" element
+  Then I see no difference for "Section 1: About You"
+   And I press the first "button#next" element
+
+  # Section 2
+  Then I wait until element "h3" contains text "Section 2: Your School"
+   And I press keys "nonexistent" for element "#school input"
+
+  # School: select other and enter manual data
+  Then I wait until element ".VirtualizedSelectOption:contains('Other school not listed below')" is visible
+   And I press ".VirtualizedSelectOption:contains('Other school not listed below')" using jQuery
+  Then I wait until element "input#schoolName" is visible
+   And I press keys "Code.org" for element "input#schoolName"
+   And I press keys "1501 4th Ave" for element "input#schoolAddress"
+   And I press keys "Seattle" for element "input#schoolCity"
+   And I select the "Washington" option in dropdown "schoolState"
+   And I press keys "98101" for element "input#schoolZipCode"
+   And I press the first "input[name='schoolType'][value='Other']" element
+  Then I press keys "Albus" for element "input#principalFirstName"
+   And I press keys "Dumbledore" for element "input#principalLastName"
+   And I press keys "socks@hogwarts.edu" for element "input#principalEmail"
+   And I press keys "socks@hogwarts.edu" for element "input#principalConfirmEmail"
+   And I press keys "5555882300" for element "input#principalPhoneNumber"
+   And I press the first "input[name='currentRole']" element
+   And I press the first "input[name='gradesAtSchool']" element
+   And I press the first "input[name='gradesTeaching']" element
+   And I press the first "input[name='gradesExpectToTeach']" element
+   And I press the first "input[name='subjectsTeaching']" element
+   And I press the first "input[name='subjectsExpectToTeach']" element
+   And I press the first "input[name='doesSchoolRequireCsLicense']" element
+   And I press the first "input[name='haveCsLicense']" element
+   And I press the first "input[name='subjectsLicensedToTeach']" element
+   And I press the first "input[name='taughtInPast']" element
+   And I press the first "input[name='previousYearlongCdoPd']" element
+   And I press the first "input[name='csOfferedAtSchool']" element
+   And I press the first "input[name='csOpportunitiesAtSchool']" element
+  Then I see no difference for "Section 2: Your School"
+   And I press the first "button#next" element
+
+  # Section 3
+  Then I wait until element "h3" contains text "Section 3: Choose Your Program"
+   And I press the first "input[name='program']" element
+   And I press the first "input[name='csdWhichGrades']" element
+   And I press the first "input[name='csdCourseHoursPerWeek']" element
+   And I press the first "input[name='csdCourseHoursPerYear']" element
+   And I press the first "input[name='csdTermsPerYear']" element
+   And I press the first "input[name='planToTeach']" element
+  Then I see no difference for "Section 3: Choose Your Program"
```

Comment:
This is good - would it be much more work to include switching the program and filling this out for all the CSP elements?

Reply:
No, that's not so bad. I'll add it (with another snapshot).